### PR TITLE
Update MCS/PCS endpoint stability designations

### PR DESCRIPTION
**Pull request by mdholloway**
* Bump css, javascript, and media-list to unstable
* Add experimental designation to mobile-html-offline-resources and talk, which had no stability designation

Phab: https://phabricator.wikimedia.org/T250505

### CODE REVIEW

**Comment by berndsi** [comment]
@mdholloway I think we could keep the talk endpoint as experimental. This one seems like that could change significantly or become obsolete with the talk page consultation project also happening.

**Comment by mdholloway** [comment]
> @mdholloway I think we could keep the talk endpoint as experimental. This one seems like that could change significantly or become obsolete with the talk page consultation project also happening.

Done

**Comment by mdholloway** [comment]
Removed the mobile-sections deprecations following further discussions about ongoing mobile-sections maintainership.

File: v1/css.yaml [hunk 1]
```diff
@@ -28,10 +28,12 @@ paths:
         The first two are the same regardless of what domain is used.
```
Comment (berndsi) [inline v1/css.yaml:28]: Probably should update this, too. Site is in "the first two". It probably meant to say the first and the third, but probably better to be more explicit here.

Comment (mdholloway) [reply v1/css.yaml]: Updated this text.

**Comment by mdholloway** [comment]
Good catch spotting that.  Should be fixed now.  For the record, for a tweak like that I don't mind if you just push a fix onto the PR.

**Pull request merged**